models/olids/base/base_olids_practitioner_in_role.sql

```
@@ -6,31 +6,35 @@
-    src.record_owner_organisation_code,
-    src.lds_datetime_data_acquired,
-    src.lds_initial_data_received_date,
+    src.lds_datetime_first_acquired,
+    src.lds_datetime_update_acquired,
     src.lds_is_deleted,
-    src.lds_start_date_time,
+    src.lds_start_datetime,
     src.lds_lakehouse_date_processed,
     src.lds_lakehouse_datetime_updated
 FROM {{ source('olids_common', 'PRACTITIONER_IN_ROLE') }} src
+INNER JOIN {{ ref('base_olids_organisation') }} org
```

SQLFluff (4.2.1)

[error] Implicit/explicit aliasing of table.

(AL01)